packages/client/wallets/smart-wallet/src/blockchain/wallets/account/cache.ts

```diff
@@ -0,0 +1,43 @@
+import { keccak256, toHex } from "viem";
+
+import { UserParams } from "../../../types/params";
+import { SmartWalletConfigSchema } from "../../../types/schema";
+import type { SmartWalletConfig } from "../../../types/service";
+
+export class AccountConfigCache {
```

Comment:
I wonder if we should generalize this to be usable by other objects besides account configs. You'd have a better sense on how useful this would be

Reply:
Next time we need to cache something I think this would make a lot of sense, but this PR brings network requests that are within our control (ie. not within the ZD SDK) to 0 for the average request, so not needed yet!